packages/dds/tree/src/feature-libraries/modular-schema/fieldChangeHandler.ts

```diff
@@ -157,6 +203,31 @@ export type NodeChangeDecoder = (change: JsonCompatibleReadOnly) => NodeChangese
  */
 export type IdAllocator = () => ChangesetLocalId;
 
+/**
+ * @alpha
+ */
+export enum CrossFieldTarget {
+	Source,
+	Destination,
+}
+
+/**
+ * @alpha
+ */
+export interface CrossFieldManager<T = unknown> {
+	getOrCreate: (
```

Comment:
Last we chatted about this interface I think you were considering refactoring it to clarify the contract with repspect to how the functions impact registered dependencies. Are you still hoping to do that? If not it would be good to put that information in doc comments.

Reply:
I'm not going to change them in this PR, but I'll add comments.